### PR TITLE
Robot Accountant

### PR DESCRIPTION
**Pull request by dariia-romanova**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,26 @@
...
 function makeRobotAccountant() {
-  // write code here
+  let count = 3;
+  let evenCount = 0;
+
+  const getSum = (num1) => {
+    const adding = (num2) => {
+      count--;
+
+      if (count < 0 && evenCount % 2 === 0) {
```
Comment (dmtrhrytsak) [inline src/makeRobotAccountant.js:16]: You could implement this glitching functionality with only one variable - `count`)

**Review by dmtrhrytsak — APPROVED** [review]
It counts money, and that's all that matters) Good job!

File: src/makeRobotAccountant.js (hunk 1 above)
Comment (dariia-romanova) [reply src/makeRobotAccountant.js:16]: You're right)

**Review by ihor-jpeg — APPROVED** [review]
well done!

**Pull request closed without merging**